bundle/manifests/mongodb-atlas-kubernetes.clusterserviceversion.yaml

```diff
@@ -547,12 +547,12 @@ metadata:
     operators.operatorframework.io/project_layout: go.kubebuilder.io/v4
     repository: https://github.com/mongodb/mongodb-atlas-kubernetes
     support: support@mongodb.com
-    containerImage: mongodb/mongodb-atlas-kubernetes-operator:2.9.1
+    containerImage: docker.io/mongodb/mongodb-atlas-kubernetes-operator:v2.10.0
   labels:
     operatorframework.io/arch.amd64: supported
     operatorframework.io/arch.arm64: supported
     operatorframework.io/os.linux: supported
-  name: mongodb-atlas-kubernetes.v2.9.1
+  name: mongodb-atlas-kubernetes.v0.0.0
```

Comment:
this seems wrong

Reply:
this is just a cp command, I guess the releases folder is also wrong?
checking

Reply:
It would be good to understand why

Reply:
2.10 has a false release, and the tag has been fixed a couple of times already. Still not sure that explains it